src/main/java/appeng/hotkeys/LocatingServicesImpl.java

```diff
@@ -0,0 +1,59 @@
+package appeng.hotkeys;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import appeng.api.hotkeys.LocatingService;
+import appeng.api.hotkeys.LocatingServices;
+import appeng.core.definitions.AEItems;
+import appeng.core.definitions.ItemDefinition;
+import appeng.items.tools.powered.PortableCellItem;
+
+/**
+ * registers {@link LocatingService}
+ */
+public class LocatingServicesImpl {
```

Comment:
I think it would be nice to expose the ids we use in the API, so that e.g. a portable DISK could be opened with the `portable_item_cell` keybinding.